modes/basic-dev-mode/src/index.ts

```diff
@@ -91,14 +91,17 @@ function modeFactory({ modeConfiguration }) {
 
       toolGroupService.createToolGroupAndAddTools('default', tools);
 
+      const buttonSectionId = 'primary';
+      const extensionsToolbarSection = toolbarService.getButtonSection(buttonSectionId) ?? [];
       toolbarService.addButtons(toolbarButtons);
-      toolbarService.createButtonSection('primary', [
+      toolbarService.createButtonSection(buttonSectionId, [
         'MeasurementTools',
         'Zoom',
         'WindowLevel',
         'Pan',
         'Layout',
         'MoreTools',
+        ...extensionsToolbarSection.map(button => button.id),
```

Comment:
i'm not sure what it means to have toolbar section added to "tools"

Reply:
This line adds the button ids that were added previously by the extension but that will be removed by the mode if we don't re-add them.

Reply:
Related to https://github.com/ImagingDataCommons/ViewersV3/pull/67

Reply:
Do it in toolbarservice, not in the mode